ts/frontend/src/app/views/benchmarks-detail/benchmarks-detail.view.ts

```diff
@@ -82,4 +85,11 @@ export class BenchmarksDetailView implements OnInit {
       }, 10000)
```

Comment:
https://en.wikipedia.org/wiki/Magic_number_(programming) -> extract to named constant.